build.gradle.kts

```diff
@@ -76,6 +86,7 @@ tasks {
     }
 
     withType<ShadowJar> {
+        mergeServiceFiles()
```

Comment:
Tror denne må til i nyeste versjon av flyway, ref https://nav-it.slack.com/archives/C60FFACN5/p1705585563248729
Bruker også `flyway-database-postgresql` i stedet for bare `flyway-core` nå.